src/resources/views/crud/inc/column_anchors.blade.php

```diff
@@ -0,0 +1,25 @@
+@php
+
+    if($column['anchor'] !== false) {
+        $related_model_key = $related_model_key ?? null;
+        $column['anchor']['href'] = isset($column['anchor']['href']) ? (is_callable($column['anchor']['href']) ? $column['anchor']['href']($crud, $column, $entry, $related_model_key) : ($column['anchor']['href'] ?? '')) : '';
```

Comment:
This is a great start! Let's make it even more general!

I don't think we should limit the developer to these attributes (```href```, ```class``` and ```target```). What if he wants to do something like
- ```style```, even if it's a little dirty, just this once;
- ```id``` 
- ```data-value``` because he does something with JS in ```list.js```

He can't, because we limit him to these few attributes. But... we don't have to. It doesn't hurt us to support _all possible attributes_. Right?

Similarly to how we do it for for ```wrapperAttributes``` in field, we could allow the developer to specify ANY attribute, and we'd place it in HTML, with ```@foreach ($column['anchor'] as $attributeKey => $attributeValue)```

All value of the attribute, as passed in the ```addColumn()``` statement, can be:
- (a) a string, which we then echo;
- (b) a closure, which we run to get the result, then echo;

That way, the developer gets MAXIMUM flexibility.

Reply:
Like a charm. :ok_hand: